[custom-functions-runtime] Update CustomFunctionsRuntime 1.3 divisionByZero error code comment, per feedback

alison-mk opened the pull request:
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: This change addresses feedback from PR #47866. 
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

akrantz reviewed (approved):
Well that's an easy way to address it!

alison-mk commented:
Note for reviewers: This change addresses feedback from PR #47866. After consulting the original author of this comment (the comment was originally included in the docs, by another writer), I decided to remove the comment from both the docs and this type definition file. The other error code comments in this file define the error codes themselves, and do not include suggestions for how to write the error handler scripts, so it's out of context to only mention it for this error code.
